### ISSUE TITLE
Bullets on archive descriptions are very far left in Firefox only

### ISSUE THREAD

**Issue by pollecuttn** [wellcomecollection/wellcomecollection.org#9418]
**What**

minor, cosmetic note: the bullets in the archive description on Audrey Amiss have gone very far left in Firefox only.

Top: Safari
Bottom: Firefox
![image](https://user-images.githubusercontent.com/1446072/224045939-b6e79584-3c42-4408-ab43-39ea33a698d3.png)

Slack thread: https://wellcome.slack.com/archives/C8X9YKM5X/p1678143425263899

**Where**

https://wellcomecollection.org/works/dw67bv4q

To do:
- [ ] Check if we want to change this given that things look different in different browsers
- [ ] Check if this shows up elsewhere on works other than archives pages

Done when
Bullets in Firefox aren't so far left

**Comment by gestchild** [comment]
This is the result of the html we receive in the [JSON response from the catalogue API](https://api.wellcomecollection.org/catalogue/v2/works/dw67bv4q?include=identifiers%2Cimages%2Citems%2Csubjects%2Cgenres%2Ccontributors%2Cproduction%2Cnotes%2CformerFrequency%2Cdesignation%2Cparts%2CpartOf%2CprecededBy%2CsucceededBy%2Clanguages%2Choldings)

The list items are not wrapped in a`<ul>` tag and this is handled differently by different browsers:

![Image](https://user-images.githubusercontent.com/6051896/226976638-67661a6b-61a4-49e5-bba6-56cb59e990fa.png)

If I use developer tools to wrap the list items, then Firefox (and Chrome) both show the list indented in the same way:

![Image](https://user-images.githubusercontent.com/6051896/226976575-6299f2e2-305c-47b0-a4a2-cbfae089340f.png)

In answer to the to do points above

- We should change this if we can as it is not purely cosmetic, the html isn't correct
- It's not related to archive pages, but the html returned in from the catalogue API, so could potentially happen in other places.

**Comment by alexwlchan** [comment]
I don't think we should be fixing this ourselves; this HTML comes directly from Collections – I'd raise it with them and ask if they can fix it.

**Comment by pollecuttn** [comment]
I've messaged about this - the person I want to talk to is currently on leave.
